### ISSUE TITLE
Panicked/petrified/... person cannot be interacted with. You should be able to attempt to.

### ISSUE THREAD

**Issue by PatrikLundell** [CleverRaven/Cataclysm-DDA#67964]
### Is your feature request related to a problem? Please describe.

I finally encountered a Petrified Person who wasn't promptly killed by zombies and found that an attempt to talk to it failed completely: I was only allowed to talk to the companion.

It's immersion breaking to not be able to try to talk to what the game tries to present as a person.

### Solution you would like.

You should be able to talk to this kind of creature and be "answered" with silence, gibberish, flight, etc.

I wouldn't recommend this option to appear for zombies due to their sheer numbers: trying to talk to your companion when faced with a horde would result in being faced with a massive list of useless alternatives.

I guess it could be handled by an optional property on the creature type that specified a set of dialog response options.

### Describe alternatives you have considered.

Ignore it as it's not exactly critical.

**Comment by GuardianDll** [comment]
does it mean `copy-from` doesn't handle `chat_topics`? 
https://github.com/CleverRaven/Cataclysm-DDA/blob/6daa9579da0448c84c1105ba1e5e6bc9cf57567b/data/json/monsters/civilians.json#L3-L13

**Comment by Maleclypse** [comment]
> does it mean `copy-from` doesn't handle `chat_topics`?
> 
> https://github.com/CleverRaven/Cataclysm-DDA/blob/6daa9579da0448c84c1105ba1e5e6bc9cf57567b/data/json/monsters/civilians.json#L3-L13

I believe that is what it means.

**Comment by CatchScratch** [comment]
Hi all. I want to get familiar with the CDDA code base so I took a quick look at this issue.

![petrified_person_dialogue](https://github.com/CleverRaven/Cataclysm-DDA/assets/42708799/83c965a8-fca9-412a-bdbc-19389a3935fb)

Using master, I spawned a panicked person and petrified person. I was able to talk with both of them. They both seem to have the TALK_CIVILIAN_PANIC talk_topic. 

![companion_dialogue](https://github.com/CleverRaven/Cataclysm-DDA/assets/42708799/92bbea4c-e101-47bd-8b1e-cc669add4257)

I wanted to make sure it wasn't an issue with having a companion nearby. I was able to talk to the petrified/panicked person with a friendly NPC near me.

TLDR: I cannot reproduce this issue.
Does that help?

**Comment by PatrikLundell** [comment]
I believe it does help to establish that it's not a general problem. I tried to teleport back to the location and make a new attempt to talk to the creature, but didn't find it, but rather a fat zombie, so I assume it somehow managed to get itself killed (alone in an apartment in a supposedly cleared apartment block) and rose again, so I'm unable to retry it.
